### PR TITLE
Use node-sass 3.x

### PR DESCRIPTION
**Pull request by hypoalex**
This should enable sass-brunch to work on SunOS with some additional downstream fixes. It definitely seems to work fine on Linux.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "test": "mocha"
   },
   "dependencies": {
-    "node-sass": ">=1.0.1 <3.0.0",
```
Comment (es128) [inline package.json]: The version should not be open-ended - future major bumps can break compatibility. Also the range should include all the versions the plugin is compatible with, giving users the ability to stick with an older one if they need.

`>=1.0.1 <4.0.0` would probably be appropriate if you've confirmed it's working with 3.x

**Comment by hypoalex** [comment]
Alright, this actually works now. I was using Ruby and not node-sass, I had forgotten about it until I actually tried to deploy on Illumos. This removes support for .sass files, don't we all use .scss? I propose calling this 2.x unless there's some compelling reason to support .sass, and I can't think of one.

Another improvement would be to only use compression on production builds. Thoughts?

Also, why use Ruby? node-sass 3.x is WAY faster.

19 Jul 12:35:01 - info: compiled 4 files into 3 files, copied 4 in 2738ms
VERSUS RUBY
19 Jul 09:51:15 - info: compiled 4 files into 3 files, copied 4 in 21606ms

**Comment by ryansobol** [comment]
`node-sass` upgraded to 3.x? Yes please. What's the hold up here? Summer vacation?

**Comment by es128** [comment]
@ryansobol this PR cannot be merged as is for a number of reasons. Feel free to submit a cleaner one that does the upgrade without dropping .sass support.

**Comment by hypoalex** [comment]
I'll look into retaining .sass support this week, I'm not totally sure
offhand how much effort this would be.

Please let me know about any other issues and I'll look into getting this
merged.

**Comment by es128** [comment]
Other issues I see are the merge conflicts, package version bump should be kept out of the PR, and lots of noisy commits that ought to get squashed.

Thanks!

**Comment by paulmillr** [comment]
@es128 my browser went crazy and accidentally deleted your comment — sorry for that.

It started with:

>Other issues I see are the merge conflicts, package version bump should be kept out of the PR, and lots of noisy commits that ought to get squashed. …

**Comment by es128** [comment]
np - I think what you pasted was the entire comment

**Comment by fire** [comment]
Was there any progress on this? My testbed just ran into problems with brunch and .sass support on smartos.

**Comment by hayesgm** [comment]
I created a fork of this that adds support for sass. We can work that into this PR or create a separate one.

https://github.com/leadtrader/sass-brunch/pull/1

**Comment by es128** [comment]
Closing in favor of #86

**Pull request closed without merging**